src/applications/claims-status/tests/reducers/serialize.unit.spec.jsx

```diff
@@ -136,4 +152,48 @@ describe('Claim Serializer', () => {
     expect(trackedItem3.documents.length).to.be.equal(2);
     expect(trackedItem3.documents.map(d => d.id).sort()).to.eql([3, 4]);
   });
+
+  // <tempfix>
+  // See notes on PMR Pending in serialize.js
+  it('should manually override the status of PMR Pending requests', () => {
+    const claim = {
+      id: 1,
+      type: 'claim',
+      attributes: {
+        trackedItems: [
+          {
+            id: 1,
+            displayName: 'PMR Pending',
+            status: 'NEEDED_FROM_YOU',
+          },
+        ],
+      },
+    };
+    const serializedClaim = serializeClaim(claim);
+    const trackedItem1 = serializedClaim.attributes.trackedItems.find(
+      d => d.id === 1,
+    );
+    expect(trackedItem1.status).to.eql('NEEDED_FROM_OTHERS');
+  });
+  // </tempfix>
+
+  it('should replace the display name of some types of tracked items', () => {
+    const claim = {
+      id: 1,
+      type: 'claim',
+      attributes: {
+        trackedItems: [
+          {
+            id: 1,
+            displayName: 'PMR Pending',
+          },
+        ],
+      },
+    };
+    const serializedClaim = serializeClaim(claim);
+    const trackedItem1 = serializedClaim.attributes.trackedItems.find(
+      d => d.id === 1,
+    );
+    expect(trackedItem1.displayName).to.eql('Private Medical Record');
+  });
```

Comment:
These tests are basically doing the same thing, but one tests the `status` while the other tests the `displayName`. Since testing the status is temporary, I kept them separate so that we can just remove that whole test case down the road.